packages/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/fresco/FrescoModule.java

```diff
@@ -129,6 +130,15 @@ public void initialize() {
 
     ReactApplicationContext reactContext = getReactApplicationContext();
     reactContext.addLifecycleEventListener(this);
+
+    reactContext.executeRuntimeExecutor(
+        new ReactContext.ReactRuntimeExecutor() {
+          @Override
+          public void execute(RuntimeExecutor runtimeExecutor) {
+            // do something
+          }
+        });
+
```

Comment:
```suggestion
```
It doesn't do anything 🤷

Reply:
oh whoops this was my testing code hehe

Reply:
We should use a different module for this test. 

At meta: FrescoModule is eagerly initialized when the ReactInstance is created. 

So when this initialize method is executed, the ReactContext and the ReactHost don't have access to the ReactInstance reference (and hence the runtime executor reference), because it's being created.

That's probably why you had to create an async api to access the RuntimeExecutor?